adapters/rubicon/rubicon_test.go

```diff
@@ -26,6 +27,12 @@ type rubiAppendTrackerUrlTestScenario struct {
 	expected string
 }
 
+type rubiSetImpNativeTestErrorScenario struct {
```

Comment:
Since this struct is specifically for `setImpNative` test cases in `TestSetImpNative`, I'd recommend just defining this inline on L266 like:
```
testScenarios := []struct {
		request       string
		impNative     map[string]interface{}
		expectedError error
	} {
		{
			request:       "{}",
			impNative:     map[string]interface{}{"somekey": "someValue"},
			expectedError: fmt.Errorf("unable to find imp in json data"),
		},
	}
```